src/ger.cc

```diff
@@ -5,118 +5,138 @@
 
 #include "blas/fortran.h"
 #include "blas.hh"
+#include "blas_internal.hh"
 
 #include <limits>
 
 namespace blas {
 
-// =============================================================================
-// Overloaded wrappers for s, d, c, z precisions.
+//==============================================================================
+namespace internal {
 
-// -----------------------------------------------------------------------------
-/// @ingroup ger
-void ger(
-    blas::Layout layout,
-    int64_t m, int64_t n,
-    float alpha,
-    float const *x, int64_t incx,
-    float const *y, int64_t incy,
-    float       *A, int64_t lda )
+//------------------------------------------------------------------------------
+/// Low-level overload wrapper calls Fortran, float version.
```

Comment:
Unless I misunderstand it, I think this says it is the `float` version, but the code is considering `std::complex<float>`.
But I have the feeling it does not exist maybe.
Anyhow, I would say it is missing the `float` and `double` version of `ger` or the comments should state `std::complex<float>` as it is for `geru`.

Reply:
Fixed.